bitnami/keycloak/values.yaml

```diff
@@ -313,7 +313,10 @@ keycloakConfigCli:
     helm.sh/hook-weight: "5"
   ## @param keycloakConfigCli.command Command for running the container (set to default if not set). Use array form
   ##
-  command: []
+  command:
```

Comment:
The idea of `command` is to override the default one. If what you are proposing should be the default command, then I would move it directly to the yaml, so it looks like this.

```
          {{- if .Values.keycloakConfigCli.command }}
          command: {{- include "common.tplvalues.render" (dict "value" .Values.keycloakConfigCli.command "context" $) | nindent 12 }}
          {{- else }}
              - java
              - -jar
              - '{{- printf "/opt/bitnami/keycloak-config-cli/keycloak-config-cli-%s.jar" .Chart.AppVersion }}'
          {{- end }}
```